src/CloudFlareZoneInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareZoneInterface.
+ */
+
+namespace Drupal\cloudflare;
+use CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings;
+
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudFlareZoneInterface {
+
+  /**
+   * Updates a zone's settings from CloudFlare's API.
+   *
+   * @param \CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings $zone_settings
+   *   The updated settings object.
+   */
+  public function updateZoneSettings(ZoneSettings $zone_settings);
+
+  /**
+   * Gets the zone's settings from CloudFlare's API.
+   *
+   * @return \CloudFlarePhpSdk\ApiTypes\Zone\ZoneSettings|NULL
```

Comment:
Style: ``null`` instead of ``NULL`` in your ``@return``... I've seen it like this all over core, ``NULL`` everywhere else.